ocp_resources/node_network_configuration_policy.py

```diff
@@ -102,16 +104,18 @@ def __init__(
         self.res = None
         self.ipv4_ports_backup_dict = {}
         self.ipv6_ports_backup_dict = {}
+        self.nodes = self._nodes()
+
+    def _nodes(self):
         if self.node_selector:
-            self._node_selector = {
-                f"{self.ApiGroup.KUBERNETES_IO}/hostname": self.node_selector
-            }
-            self.nodes = list(Node.get(dyn_client=self.client, name=self.node_selector))
-        else:
-            self._node_selector = {
-                f"node-role.{self.ApiGroup.KUBERNETES_IO}/worker": ""
-            }
-            self.nodes = list(Node.get(dyn_client=self.client))
+            return list(Node.get(dyn_client=self.client, name=self.node_selector))
+        if self.node_selector_labels:
+            label_key, label_value = list(self.node_selector_labels.items())[0]
+            return list(
+                Node.get(
+                    dyn_client=self.client, label_selector=f"{label_key}={label_value}"
```

Comment:
why not support multiple labels? 
",".join(["f{label_key}={label_value}" for label_key, label_value in self.node_selector_labels.items()])

Reply:
+1